pkgs/development/python-modules/lap/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  cython,
+  fetchPypi,
+  numpy,
+  pytestCheckHook,
+  python-utils,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "lap";
+  version = "0.5.12";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    pname = "lap";
+    inherit version;
+    hash = "sha256-VwtBTqeubAS9SdDsjNrB3FY0c3dVeE1E43+fZourRP0=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeBuildInputs = [ cython ];
+
+  dependencies = [
+    numpy
+    python-utils
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "lap" ];
+  preCheck = ''
+    cd "$out"
+  '';
```

Comment:
This is due to https://github.com/NixOS/nixpkgs/issues/255262 right? Could you please add a link to this issue as a comment?